sklearn/feature_selection/tests/test_rfe.py

```diff
@@ -182,6 +182,12 @@ def test_scorer(estimator, X, y):
     X_r_sparse = rfecv_sparse.transform(X_sparse)
     assert_array_equal(X_r_sparse.toarray(), iris.data)
 
+    # Verifying that steps < 1 don't blow up.
+    rfecv_sparse = RFECV(estimator=SVC(kernel="linear"), step=.2, cv=5)
+    X_sparse = sparse.csr_matrix(X)
+    rfecv_sparse.fit(X_sparse, y)
+    X_r_sparse = rfecv_sparse.transform(X_sparse)
```

Comment:
Added in an assertion against the data, but I'm not sure if that's what you meant.